packages/js/src/generators/setup-verdaccio/generator.ts

```diff
@@ -50,6 +51,8 @@ export async function setupVerdaccio(
     });
   }
 
+  addLocalRegistryScripts(tree);
```

Comment:
We shouldn't always add the localregistry scripts right?

Reply:
changed